src/components/ScanQRModal/ScanQRWrapper/index.tsx

```diff
@@ -40,7 +41,7 @@ export const ScanQRWrapper = ({ handleScan }: Props): ReactElement => {
         onClick={() => openQrModal()}
         role="button"
         src={QRIcon}
-        testId="qr-icon"
+        testId={testId || 'qr-icon'}
```

Comment:
Can you not get this by role? Just in case we change anything in the future, the test will still pass because `testId` will remain the same.